src/ussd_community.js

```diff
@@ -247,9 +255,9 @@ go.app = function() {
                                 return 'state_save_identities';
                             }
 
-                            return go.utils_project
-                                .check_is_subscribed(
-                                    self.im, contact.id, 'public.mother')
+                            return sbm
+                                .is_identity_subscribed(
+                                    contact.id, [/public\.mother/, /prebirth\.mother/, /postbirth\.mother/])
                                 .then(function(subscribed) {
                                     if (!subscribed || subscribed == 'no_active_subs_found') {
```

Comment:
`subscribed` will now be a boolean, that function won't return a string, so we don't need this additional check.